Add Helm migration instructions (#3872)
elastic/cloud-on-k8s: docs/operating-eck/installing-eck.asciidoc

----
helm show values elastic/eck-operator
----

[float]
[id="{p}-migrate-to-helm"]
=== Migrate an existing installation to Helm

> If you have an operator installed using the `all-in-one.yaml` method, you can migrate to Helm by running the following script, which is also available for download from the link:{eck_github}/tree/{eck_release_branch}/deploy[ECK source repository]. It should preserve your existing Elastic applications deployed using the old operator. 

Reviewer: ```suggestion
If you have an operator installed using the `all-in-one.yaml` method, you can migrate to Helm by running the following script, that you can also download from the link:{eck_github}/tree/{eck_release_branch}/deploy[ECK source repository]. It should preserve your existing Elastic applications deployed using the old operator. 
```
